### DISCUSSION TITLE
제페토 스튜디오의 package 파일을 등록시 월드 빌드 실패

### DISCUSSION THREAD

**Discussion by novakjh1234** [naverz/zepeto-studio-kor#1196] [4. General]
안녕하세요. 월드 빌드 실패 오류가 계속 떠서 문의드립니다.
zepeto package 파일의 용량은 약 850메가입니다.
 계속하여 오류가 발생해서 조언 좀 부탁드립니다.
 [dl_zepeto_evangelist@naverz-corp.com](mailto:dl_zepeto_evangelist@naverz-corp.com) 메일로 패키지 파일을 보내드립니다.
부탁드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 보내주신 메일 확인하였습니다.
내부에서 테스트 후 답변 드리겠습니다.
감사합니다.
